composer.json

```diff
@@ -67,5 +68,10 @@
         "branch-alias": {
             "dev-master": "2.0.x-dev"
         }
+    },
+    "scripts": {
```

Comment:
No magic involved. Take a look at https://github.com/symfony/symfony-standard/blob/6832c7baae2d757b12bbd5342aea2993c21ee9eb/composer.json#L30-L44

Yes, we probably do not need a `Command`.